Gruntfile.js

```diff
@@ -74,5 +88,12 @@ module.exports = function(grunt) {
   }
 
   grunt.registerTask("dev", ["connect", "watch"]);
-  grunt.registerTask("test", ["connect", "saucelabs-custom"]);
+  grunt.registerTask("test-locally", ["connect", "jasmine"]);
+
+  var secureEnvVars = process.env.TRAVIS_SECURE_ENV_VARS;
+  if (secureEnvVars === undefined || secureEnvVars === "true") {
```

Comment:
This is weird, how come we detect if we're inside the Travis environment by either having the TRAVIS_SECURE_ENV_VARS var as `undefined` or `"true"`. Do you have a relevant link for this finding?

Either way, if this is the only way to detect consistently, you should create a separate function that abstracts the ugliness. E.g. `isRanByTravis()`